x-pack/plugins/observability_solution/infra/public/components/asset_details/tabs/anomalies/anomalies.tsx

```diff
@@ -20,16 +20,15 @@ export const Anomalies = () => {
   const { isActiveTab } = useTabSwitcherContext();
   const { request$ } = useRequestObservable();
   const { getParsedDateRange } = useDatePickerContext();
-  const { asset, overrides } = useAssetDetailsRenderPropsContext();
+  const { asset, renderMode } = useAssetDetailsRenderPropsContext();
   const { kibanaVersion, isCloudEnv, isServerlessEnv } = useContext(KibanaEnvironmentContext);
-  const { onClose = () => {} } = overrides?.anomalies ?? {};
 
   const parsedDateRange = useMemo(() => getParsedDateRange(), [getParsedDateRange]);
 
   return (
     <div ref={ref}>
       <AnomaliesTable
-        closeFlyout={onClose}
+        closeFlyout={renderMode.mode === 'flyout' ? renderMode.closeFlyout : () => {}}
```

Comment:
Apparently, the `anomalies` attribute from `overrides` isn't used anywhere. How about removing it from the type definition?

Reply:
Oh, I missed that one. I'll take care of it.

Reply:
`anomalies` removed from type definition. No usage.